Sources/SwifterSwift/MapKit/MKMapViewExtensions.swift

```diff
@@ -44,7 +44,25 @@ public extension MKMapView {
 
         return annotationView
     }
-
+    
+    /// SwifterSwift: Zooms in on multiple mapView coordinates.
+    ///
+    /// - Parameters:
+    ///   - coordinates: Gets the array of type CLLocationCoordinate2D.
+    ///   - meter: Takes the type of Double zoom.
+    ///   - insets: Gives space to the CGFloat type screen.
+    ///   - animated: The animation control takes the Boolean value.
+    @available(iOS 9.0, *)
+    func multipleCoordinateZoom(for coordinates: [CLLocationCoordinate2D], meter: Double, insets: CGFloat, animated: Bool) {
+        if coordinates.count == 1 {
```

Comment:
I think add at the start of the function
```Swift
guard !coordinates.isEmpty else { return }
```

Reply:
But non option still can be empty array. I think @guykogus meant that we should handle that empty array case.

Reply:
Yes you are right 😊